File: GDocs.Site/src/app/saida-material-nota-fiscal/shared/models/CienciaEhHistoricoResponseModel.ts
import { CienciaUsuarioAprovacaoModel } from "./ciencia-usuario-aprovacao-model";
import { HistoricoProrrogacaoNotaFiscalModel } from "./HistoricoProrrogacaoNotaFiscalModel";
import { SaidaMaterialNotaFiscalCienciaModel } from "./saida-material-nota-fiscal-ciencia-model";

export class CienciaEhHistoricoResponseModel{
  public solicitacaoCiencia: SaidaMaterialNotaFiscalCienciaModel;
  public historicoProrrogacoes:HistoricoProrrogacaoNotaFiscalModel;
  public cienciaUsuarioAprovacao:CienciaUsuarioAprovacaoModel[];

}

File: GDocs.Api/src/Infra/CrossCutting/Models/SaidaMaterialNotaFiscal/CienciaEhHistoricoResponseModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ICE.GDocs.Infra.CrossCutting.Models.SaidaMaterialNotaFiscal
{
    public class CienciaEhHistoricoResponseModel
    {
        public SaidaMaterialNotaFiscalCienciaModel SolicitacaoCiencia { get; set; }
        public IEnumerable<HistoricoProrrogacaoNotaFiscalModel> HistoricoProrrogacoes { get; set; }
        public IEnumerable<CienciaUsuarioAprovacaoModel> CienciaUsuarioAprovacao { get; set; }
        
    }
}
